pkg/csi/cinder/controllerserver.go

```diff
@@ -73,6 +73,9 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 	if len(volumes) == 1 {
 		resID = volumes[0].ID
 		resAvailability = volumes[0].AZ
+		resSize = volumes[0].Size
+
+		glog.V(4).Infof("Return created volume %s in Availability Zone: %s of size %d GiB", resID, resAvailability, resSize)
```

Comment:
It Would be better if we also log, that volume already exist.
glog.V(4).Infof("Volume %s  exists with same name in Availability Zone: %s  of  size %d GiB, resID, resAvailability, resSize)